static/js/index.js

```diff
@@ -1,12 +1,16 @@
-// Initialize Embla carousel
-const emblaRoot = document.querySelector('.embla');
-const emblaViewport = emblaRoot?.querySelector('.embla__viewport');
-const prevBtn = emblaRoot?.querySelector('.embla__prev');
-const nextBtn = emblaRoot?.querySelector('.embla__next');
-const options = { loop: true };
+// Initialize Embla carousel. 
+const emblaRoot = document.querySelectorAll('.embla');
 
 if (emblaRoot) {
-  const carousel = EmblaCarousel(emblaViewport, options);
-  prevBtn.addEventListener('click', carousel.scrollPrev, false);
-  nextBtn.addEventListener('click', carousel.scrollNext, false);
-}
\ No newline at end of file
+  emblaRoot?.forEach((root) => {
```

Comment:
Now that there are 2 carousels on the same, page, the JavaScript has been updated so that separate options may be set for each one. Although the `align` option should always be start to ensure the carousel begins with the first slide that we set. Currently, the options for both carousels are the same but adding this flexibility now will make it easier if we want to make changes.